Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_ErrorPageView.1.txt

```diff
@@ -9,9 +9,9 @@
     <title>Swift Package Index</title>
     <meta name="twitter:title" content="Swift Package Index"/>
     <meta name="og:title" content="Swift Package Index"/>
-    <meta name="description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,482 packages and 36,483 versions."/>
-    <meta name="twitter:description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,482 packages and 36,483 versions."/>
-    <meta name="og:description" content="The Swift Package Index is the place to find the best Swift packages. Indexing 2,482 packages and 36,483 versions."/>
+    <meta name="description" content="The Swift Package Index is the place to find the best Swift packages."/>
+    <meta name="twitter:description" content="The Swift Package Index is the place to find the best Swift packages."/>
+    <meta name="og:description" content="The Swift Package Index is the place to find the best Swift packages."/>
```

Comment:
The error page is a page where we lose the `meta` but we're probably not linking to those 😆